Update dist names in core docs 3.0.0 - closes #613 (#618)
LiskHQ/lisk-docs: modules/ROOT/pages/setup/commander.adoc

:url_core_2_setup: master@lisk-core::setup/commander.adoc

> Setup and manage your Lisk node conveniently with the Lisk Commander application.

Reviewer: ```suggestion
Setup and manage your Lisk Core node conveniently with Lisk Commander.
```

Reviewer: Please see my suggestion above

To complete the installation some prerequisites need to be fulfilled.
If you have already performed these, please proceed to the <<install, Installation>> chapter.

> === Install Lisk Commander application

Reviewer: ```suggestion
=== Install Lisk Commander
```

Reviewer: Please see my suggestion above

> Please access the Lisk Commander application docs and follow the xref:{url_sdk_commander_setup}[installation instructions].

Reviewer: ```suggestion
Please access the Lisk Commander docs and follow the xref:{url_sdk_commander_setup}[installation instructions].
```

> Once the Lisk Commander appliction is set up on your system, it is possible to use the commands below to set up and manage your Lisk Core node.

Reviewer: ```suggestion
Once Lisk Commander is set up on your system, it is possible to use the commands below to set up and manage your Lisk Core node.
```

| `--release-url` | `-r`
| `--release-url=https://my-custom.url/testnet/2.1.0-rc.0/lisk-2.1.0-rc.0-Linux-x86_64.tar.gz`
| Define a custom URL for downloading the new Lisk Core application script.
It is also necessary to have the checksum for the TAR file, with the format 2.0.0-Linux-x86_64.tar.gz.SHA256 for the Commander to check the integrity of the application.

> Please refer to the {url_binary_download}[application release^] for the file naming convention.

Reviewer: ```suggestion
Please refer to the {url_binary_download}[Lisk Core application releases^] for the file naming convention.
```
